src/formatDate.js

```diff
@@ -51,6 +51,54 @@
 
 function formatDate(date, fromFormat, toFormat) {
   // write code here
+  const fromSeperator = fromFormat.pop();
+  const dateArray = date.split(fromSeperator);
+  let dateYear, dateMonth, dateDay;
+  const output = [];
```

Comment:
I think output can be anything. Maybe a more meaningful variable, based on its value is better.

Reply:
Done!